packages/uicore/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.160.5",
+  "version": "3.160.6",
```

Comment:
you can increase the minor version since we are adding a new prop and not modifying/fixing anything

Reply:
done